fares-data-build-tool/fdbt-netex-output
# feat: handle scheme operators in netex

# Description

-   This PR adds logic to the periodTicketNetexGenerator and periodTicketNetexHelpers to conditionally handle scheme operators.

# Testing instructions

-   Pull down this branch and the corresponding PR in the `fdbt-dev` repo (https://github.com/fares-data-build-tool/fdbt-dev/pull/31).
-   Run the `generate-validate-scheme-operator` script in the Makefile and check that the generated NeTEx is valid and contains the correct data in the correct place.

# Type of change


# Checklist:
